### DISCUSSION TITLE
Error on MacOS M1

### DISCUSSION THREAD

**Discussion by moertel187** [madMAx43v3r/chia-plotter#243] [Q&A]
Hey while trying to install on M1 Mac mini i got these errors:

        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2544:9: error: unrecognized instruction mnemonic, did you mean: cmp?
        jmp 9b
        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2546:9: error: unrecognized instruction mnemonic
        vpxor xmm0, xmm0, xmm2
        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2547:9: error: unrecognized instruction mnemonic
        vpxor xmm1, xmm1, xmm3
        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2548:9: error: unrecognized instruction mnemonic
        vpxor xmm2, xmm2, [rdi]
        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2549:9: error: unrecognized instruction mnemonic
        vpxor xmm3, xmm3, [rdi+0x10]
        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2550:25: error: unexpected token in argument list
        vmovdqu xmmword ptr [r9], xmm0
                        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2551:25: error: unexpected token in argument list
        vmovdqu xmmword ptr [r9+0x10], xmm1
                        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2552:25: error: unexpected token in argument list
        vmovdqu xmmword ptr [r9+0x20], xmm2
                        ^
/Users/Kaspersky/chia-plotter/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S:2553:25: error: unexpected token in argument list
        vmovdqu xmmword ptr [r9+0x30], xmm3

make[2]: *** [CMakeFiles/blake3.dir/lib/BLAKE3/c/blake3_avx512_x86-64_unix.S.o] Error 1
make[1]: *** [CMakeFiles/blake3.dir/all] Error 2
make: *** [all] Error 2

Thats just the end of the list.

What am i doing wrong?

**Comment by madMAx43v3r** [comment]
should be fixed in latest master?

**Reply by moertel187** [reply]
you can tell me whats the problem here?
